### DISCUSSION TITLE
I recall setting up my admin but I see ADMINISTERED BY Not set in the 'About' area

### DISCUSSION THREAD

**Discussion by jsailo** [mastodon/mastodon#19166] [Q&A] [answered]
I can login with the email and password that I provided in the `Do you want to create an admin user straight away?` prompt during the set up but I don't see the **Administration** tab on the **Preferences** page.

Also, when I log out I see the following in the **About** > **Learn more** area.

```
ADMINISTERED BY 
Not set

```
![image](https://user-images.githubusercontent.com/35369823/189643445-2f1bedb3-576c-487c-b587-522367e41f2c.png)

**Comment by jsailo** [comment, marked as answer]
OK. I resolved this. Sharing in case someone lands here. With help from https://littlefo.rest/@mashiro.

It seems the permission system has been updated and you now need to do this manually via `tootctl`.

I was told to run this:

`tootctl accounts modify super_admin --role Owner`. 

The above didn't work for me and after searching around, the following command worked. Strangely, from https://docs.soapbox.pub/legacy/administration/post-installation/ 
@ClearlyClaire Please correct me if the following command is the incorrect approach. Shamelessly tagging you as you are most responsive and are super helpful.

`RAILS_ENV=production bin/tootctl accounts modify owner_username --role Owner`

Where, **owner_username** is the username you want to set as the owner of the site.

You will now be able to access the `/admin/settings/edit` page of your site. 

However, the **ADMINISTERED BY** will still be blank or labelled as **Not set** on the about page of your site.

For this, update the Contact username field in the `/admin/settings/edit` page (pic below)

More about tootctl: [Using the admin CLI](https://docs.joinmastodon.org/admin/tootctl/).

**Contact username** field 

![image](https://user-images.githubusercontent.com/35369823/189792619-7a8b167d-2d72-4e95-a2f5-b286589bfad4.png)

**Reply by ClearlyClaire** [reply]
The “Contact username” thing is not automatically set to the admin account created in the `mastodon:setup` task. Maybe it should be.

As for the `Owner` role, you are correct. That you need to change it manually is an oversight, we forgot to change the `mastodon:setup` task when adding the new customizable role system.
